performance-storage-service/pss_project/settings/testing.py

```diff
@@ -1,4 +1,15 @@
 from .base import *
 
 DEBUG = True
-ALLOWED_HOSTS = ['incrudibles-testing.db.pdl.cmu.edu']
\ No newline at end of file
+ALLOWED_HOSTS = ['incrudibles-testing.db.pdl.cmu.edu']
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.postgresql',
+        'NAME': get_environ_value('PSS_DATABASE_NAME'),
+        'USER': get_environ_value('PSS_DATABASE_USER'),
+        'PASSWORD': get_environ_value('PSS_DATABASE_PASSWORD'),
+        'HOST': '127.0.0.1',
```

Comment:
same here

Reply:
Is the service name `timescaledb-service` or `incrudbiles-testing.pdl.whaterver:30003`? or maybe something else.

Reply:
according to the [k8s doc](https://kubernetes.io/docs/concepts/services-networking/service/#dns), both `timescaledb-service` and `timescaledb-service.performance` should work